frontends/web/src/components/settingsButton/settingsButton.module.css

```diff
@@ -1,6 +1,6 @@
 .container {
     width: 100%;
-    min-width: 200px;
+    min-width: 230px;
```

Comment:
This avoids that the red dot component goes on a newline when resizing the screen. It should not give troubles elsewhere.